pulsebot/pulse_dispatch.py

```diff
@@ -63,6 +63,8 @@ def add_changeset(self, cs):
             'desc': cs['desc'],
             'is_backout': bool(BACKOUT_RE.match(cs['desc'])),
         })
+        if cs.get('branch'):
+            self.changesets[-1]['branch'] = cs['branch']
```

Comment:
Theoretically speaking, a push could have multiple changesets all on different branches. So the branch should be attached to all changesets. Then when creating the bugzilla message, some kind of per branch aggregation should occur.

So a bugzilla message could look like:
```
Pushed by foo@bar.com on the branch foo:
https://server/repo/rev/6e4e7985aba3
Foo
https://server/repoa/rev/1234567890ab
Bar

Pushed by foo@bar.com on the branch qux:
https://server/repoa/rev/234567890abc
Qux
```

Or some other variation where branches are not repeated. Typical pushes would only have something like the first half.